pkg/trace/api/api.go

```diff
@@ -401,6 +402,17 @@ func (r *HTTPReceiver) handleStats(w http.ResponseWriter, req *http.Request) {
 		}
 	}
 	metrics.Count("datadog.trace_agent.receiver.stats_buckets", int64(len(in.Stats)), nil, 1)
+
+	if in.Env == "" {
+		in.Env = r.conf.DefaultEnv
+	}
+	in.Env = traceutil.NormalizeTag(in.Env)
+	for _, rule := range r.conf.ReplaceTags {
+		switch rule.Name {
+		case "env", "*":
```

Comment:
The version case is removed in this PR, I don't know if it's actually used

Reply:
It will automatically be applied to the `tags` var defined in the `for` loop